ee/dtr/admin/monitor-and-troubleshoot/troubleshoot-with-logs.md

```diff
@@ -96,10 +104,6 @@ queries such as:
     pk: 'cf5e8bf1197e281c747f27e203e42e22721d5c0870b06dfb1060ad0970e99ada',
     visibility: 'public' },
 ...
-
-# List problems detected within the rethinkdb cluster
-> r.db("rethinkdb").table("current_issues")
-...
 ```
 
 Indvidual DBs and tables are a private implementation detail and may change in DTR
```

Comment:
I know this has nothing to do with your changes, but it stuck out at me and we're already here: Let's go ahead and fix this typo while we're editing this file for the PR, `Indvidual` should read `Individual`.

Reply:
Right on.  Addressed with:

``
git grep -l Indvidual |xargs sed -i 's:Indvidual:Individual:'
```

Rebased. Re-pushed.